samples/esb/esb_prx_ble/Kconfig

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Enhanced ShockBurst: Receiver"
+
+config ESB_PRX_APP_LOG_LEVEL
+	int "Log level for the ESB PRX sample"
+	default 4
```

Please use the Kconfig template for this.
```suggestion
module = ESB_PRX_BLE
module-str = "ESB_PRX_BLE"
source "$(ZEPHYR_BASE)/subsys/logging/Kconfig.template.log_config"
```